### PR TITLE
Trello cal demo

### PR DESCRIPTION
**Pull request by adripedroza**

### CODE REVIEW

File: examples/demos/trelloCal.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import 'react-big-calendar/lib/addons/dragAndDrop/styles.scss'
+
+const DragAndDropCalendar = withDragAndDrop(Calendar)
+
+class TrelloCal extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      events: events,
+      displayDragItemInCell: true,
```
Comment (jkhoang313) [inline examples/demos/trelloCal.js]: we can remove this

File: examples/demos/trelloCal.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    this.moveEvent = this.moveEvent.bind(this)
+    this.newEvent = this.newEvent.bind(this)
+  }
+
+  handleDragStart = event => {
+    this.setState({ draggedEvent: event })
+  }
+
+  dragFromOutsideItem = () => {
```
Comment (jkhoang313) [inline examples/demos/trelloCal.js]: we can remove this too

File: examples/demos/trelloCal.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    return this.state.draggedEvent
+  }
+
+  onDropFromOutside = ({ start, end, allDay }) => {
```
Comment (jkhoang313) [inline examples/demos/trelloCal.js]: we can remove this too

File: examples/demos/trelloCal.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    })
+
+    this.setState({
+      events: nextEvents,
+    })
+
+    //alert(`${event.title} was resized to ${start}-${end}`)
+  }
+
+  newEvent(event) {
```
Comment (jkhoang313) [inline examples/demos/trelloCal.js]: we can remove this too

File: examples/demos/trelloCal.js [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+  render() {
+    return (
+      <DragAndDropCalendar
+        selectable
+        localizer={this.props.localizer}
+        events={this.state.events}
+        onEventDrop={this.moveEvent}
+        resizable
+        onEventResize={this.resizeEvent}
+        onSelectSlot={this.newEvent}
```
Comment (jkhoang313) [inline examples/demos/trelloCal.js]: we can remove this too

File: examples/demos/trelloCal.js [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+        defaultView={Views.MONTH}
+        defaultDate={new Date(2021, 3, 12)}
+        popup={true}
```
Comment (jkhoang313) [inline examples/demos/trelloCal.js]: remove this

File: examples/demos/trelloCal.js [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+        dragFromOutsideItem={
```
Comment (jkhoang313) [inline examples/demos/trelloCal.js]: remove this

File: examples/demos/trelloCal.js [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+          this.state.displayDragItemInCell ? this.dragFromOutsideItem : null
+        }
+        onDropFromOutside={this.onDropFromOutside}
```
Comment (jkhoang313) [inline examples/demos/trelloCal.js]: remove this

File: examples/demos/trelloCal.js [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+        handleDragStart={this.handleDragStart}
```
Comment (jkhoang313) [inline examples/demos/trelloCal.js]: remove this

**Comment by adripedroza** [comment]
all the commented items should be gone now

**Pull request closed without merging**